wordpress-mobile/WordPress-iOS
# Updated copy for No results view full screen displayed on connection issue

Fixes: #11030
We want to show the same copy when the full screen (no results view) is displayed in case of connectivity issue.

To test:
1. Run the app.
2. Go offline
3. Go over any of the full pages (stats, activity, site pages...) and see that the copy for the title and subtitle is consistent .
4. Check that the full screen copy hasn't changed (as it was before this PR) in case the screen appears due to other reason for example:
4.1. Create a new site
4.2. Go to Media.
4.3. See that the illustration and the copy are media specific.
4.4. Go to site pages.
4.5. See that the illustration and the copy are site pages specific.
4.6. Go to Drafts tab.
4.7. See that the copy is draft pages specific.
4.8. Go to blog posts
4.9. Delete the About post.
4.10. See that the illustration and the copy are Blog posts specific.
etc'

* This PR does not include the updated illustration
  Update release notes:
